src/java.base/share/classes/sun/security/util/KnownOIDs.java

```diff
@@ -351,7 +351,9 @@
     ECDH("1.3.132.1.12"),
 
     // OIW secsig 1.3.14.3.*
-    OIW_DES_CBC("1.3.14.3.2.7", "DES/CBC"),
+    OIW_DES_CBC("1.3.14.3.2.7", "DES/CBC", "DES"),
+
+    DESede("1.3.14.3.2.17", "DESede"),
```

Comment:
Please move this below before SHA-1. The items are ordered by the OIDs (within each section group).